var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -782,6 +785,7 @@ class Opencv(CMakePackage, CudaPackage):
         description="Enable -ffast-math (not recommended for GCC 4.6.x)",
     )
     variant("nonfree", default=False, description="Enable non-free algorithms")
+    variant("contrib", default=True, description="Install extra OpenCV modules")
```

Comment:
Perhaps the default should be `False`? Also, change the description to:
"Enable OpenCV contrib modules"
as "extra" is not as specific.

Reply:
As a personal rule, whenever I introduce a variant to an existing package, I try to preserve the default behavior. Currently, the `contrib` modules are always installed, so I keep it that way.